locale/fi/LC_MESSAGES/django.po

```diff
@@ -3077,6 +2895,126 @@ msgstr "Ei, vie minut takaisin"
 msgid "Reset allocations"
 msgstr "Nollaa allokoinnit"
 
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "The Paytrail Merchant ID should be a six-digit number."
+msgstr "Kaupan paytrail-tunnuksen tulisi olla kuusinumeroinen."
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Shop ID"
+msgstr "Kaupan tunnus"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Business ID"
+msgstr "Y-tunnus"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "ZIP code"
+msgstr "Postinumero"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Email address"
+msgstr "Sähköpostiosoite"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "URL"
+msgstr "Verkko-osoite"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Terms of service URL"
+msgstr "Käyttöehtojen URL"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Merchant ID"
+msgstr "Kauppiastunniste"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Merchant name"
+msgstr "Kauppiaan nimi"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Value comes from the Merchant Experience API"
+msgstr "Arvo tulee kauppiaskokemusrajapinnasta"
+
+#: tilavarauspalvelu/admin/payment_product/admin.py
```

Comment:
cf. https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/1343#discussion_r1766203958
(and multiple lines immediately below)

Reply:
Fixed